doc/psidk/book.toml.in

```diff
@@ -2,7 +2,7 @@
 language = "en"
 multilingual = false
 src = "${CMAKE_CURRENT_SOURCE_DIR}/doc/psidk/src"
-title = "psidk"
+title = "doc-sys"
```

Comment:
The intention was to make it the name of the app. Since psidk is neither the name of the app nor what is being documented (exclusively). 
But okay, I renamed it `psibase` as a generic indication as to what is being documented.